osu.Server.Queues.ScoreStatisticsProcessor/Processors/UserTotalPerformanceProcessor.cs

```diff
@@ -136,21 +139,24 @@ private static async Task updateGlobalRank(UserStats userStats, MySqlConnection
                     }, transaction);
             }
 
-            // User's 90-day rolling rank history.
-            int todaysRankColumn = await connection.QuerySingleOrDefaultAsync<int?>(@"SELECT `count` FROM `osu_counts` WHERE `name` = @todaysRankColumn", new
-            {
-                dbInfo.TodaysRankColumn
-            }, transaction) ?? 0;
-
-            await connection.ExecuteAsync(
-                $"INSERT INTO `osu_user_performance_rank` (`user_id`, `mode`, `r{todaysRankColumn}`) VALUES (@userId, @mode, @rank) "
-                + $"ON DUPLICATE KEY UPDATE `r{todaysRankColumn}` = @rank",
-                new
-                {
-                    userId = userStats.user_id,
-                    mode = dbInfo.RulesetId,
-                    rank = userStats.rank_score_index
-                }, transaction);
+            // Update user's 90-day rolling rank history for today.
+            // TODO: This doesn't need to be updated here. the osu-web graph always uses `rank_score_index` for the current day.
+            // TODO: code is left for now as it will need to be used in the future for daily processing purposes (once migrated from web-10).
```

Comment:
I know we've had this conversation already, and as stated before I agree that this update doesn't make sense, but I'm kinda not seeing why this pull *specifically* has to disable this if the entire call to `updateGlobalRank()` is if'd out already in the new command...? Which has me worried I might be misunderstanding something about how this is supposed to work.

Either way, don't care about this change as stated, just a bit confused why this came up here at all.

Reply:
I'd just rather not cause undue database load on what will be a very common query to run (~20-100 times a second for incoming `_high` scores).

I was on the fence of whether to TODO or TODO and comment out. Since you're hesitant I'll switch back to just a TODO for now and we can revisit later.

Reply:
> I'd just rather not cause undue database load on what will be a very common query to run (~20-100 times a second for incoming `_high` scores).

I figured it was likely going to be this but just wanted to make sure.

Again, I was fine with commenting it out, just mostly wondering why this came up at all :shrug: If you think it's best it's commented out I'd revert to it being commented out, you have a better handle of db perf characteristics than I do.

Reply:
nah, let's match web-10 for now and reassesss later.